dll/cpl/usrmgr/groupprops.c

```diff
@@ -9,6 +9,10 @@
 
 #include "usrmgr.h"
 
+#include "wine/debug.h"
+#include "wine/unicode.h"
```

Comment:
```suggestion
#include <wine/debug.h>
#include <wine/unicode.h>
```

And (why) is `unicode.h` actually needed?

Reply:
right